website/content/api-docs/auth/kubernetes.mdx

```diff
@@ -25,6 +25,16 @@ access the Kubernetes API.
 | :----- | :------------------------ |
 | `POST` | `/auth/kubernetes/config` |
 
+### Deprecations
+
+The following fields have been deprecated and will be removed in a future release:
+
+* `disable_iss_validation` `(bool: true)` Disable JWT issuer validation. Allows to skip ISS validation.
+
+
+* `issuer` `(string: "")` Optional JWT issuer. If no issuer is specified, then this plugin will
+
+
```

Comment:
I'd suggest moving this section below the `Parameters` section, and calling it `Deprecated Parameters`.